proposals/0195-dynamic-member-lookup.md

```diff
@@ -0,0 +1,790 @@
+# Introduce User-defined "Dynamic Member Lookup" Types
+
+* Proposal: [SE-0195](0195-dynamic-member-lookup.md)
+* Author: [Chris Lattner](https://github.com/lattner)
+* Review Manager: TBD
+* Implementation: [PR13361](https://github.com/apple/swift/pull/13361)
```

Comment:
The [status page](https://apple.github.io/swift-evolution/) requires this to be written as:

```markdown
* Implementation: [apple/swift#13361](https://github.com/apple/swift/pull/13361)
```

Reply:
I agree with your suggested change here, although I'd argue that the requirement comes from the [template](https://github.com/apple/swift-evolution/blob/master/0000-template.md), not the status page. People benefit from consistency whether they're looking at the individual proposals or the status page.